README.md

```diff
@@ -159,9 +159,14 @@ TODO:
 
 ### Accessibility
 
+From an accessibility perspective, portals behave like a single activatable element (similar to a button). As discussed in the section above, portals are not interactive and don't receive input events and focus. As a result, the embedded contents of a portal are not exposed as elements in the accessibility tree.
+
+Portals come with accessibility defaults right out of the box. Their default ARIA role is `"button"`, and they are therefore visible to screen-readers as a button by default. The portal element is tab-focusable and keyboard activatable by default (any registered `click` event handlers will be run when a user hits `<space>` or `<enter>` while focused on the element).
```

Comment:
Since focusability, especially tab-focusability, is different per browser, we shouldn't make blanket statements here. The same goes for Space or Enter, I think. So I'd suggest stating something like "The portal element is intended by focusable and keyboard activatable in the same way as a button", or something like that?

Reply:
Updated. I was trying to indicate that it respects the requirements for the button role here: https://www.w3.org/TR/wai-aria-practices-1.1/#button, but I think your suggestion gets that across too.